apps/trading/pages/markets/[marketId].page.tsx

```diff
@@ -0,0 +1,89 @@
+import { gql, useQuery } from '@apollo/client';
+import AutoSizer from 'react-virtualized-auto-sizer';
+import classNames from 'classnames';
+import { useRouter } from 'next/router';
+import { ReactNode } from 'react';
+import { Market, MarketVariables } from './__generated__/Market';
+
+// Top level page query
+const MARKET_QUERY = gql`
+  query Market($marketId: ID!) {
+    market(id: $marketId) {
+      id
+      name
+      trades {
+        id
+        price
+        size
+        createdAt
+      }
+    }
+  }
+`;
+
+const MarketPage = () => {
+  const { query } = useRouter();
+  const { data, loading, error } = useQuery<Market, MarketVariables>(
+    MARKET_QUERY,
+    {
+      variables: { marketId: query.marketId as string },
+    }
+  );
+
+  if (loading || !data) {
+    return <div>Loading...</div>;
+  }
+
+  return (
+    <>
+      {error ? (
+        <div>Something went wrong: {error.message}</div>
+      ) : (
+        <div className="h-full max-h-full grid gap-[1px] bg-black grid-cols-[300px_2fr_1fr_1fr] grid-rows-[min-content_1fr_200px]">
+          <header className="bg-white col-span-4 p-8">
+            <h1>Market: {query.marketId}</h1>
+          </header>
+          <GridChild name="Ticket">Content</GridChild>
```

Comment:
(question) non-blocking: Are we going i18n first or just english for first version?

Reply:
IMO lets leave i18n until we actually have some full translations ready to go. Open for debate though